admin-site/.gitignore

```diff
@@ -0,0 +1,22 @@
+.DS_Store
+node_modules
+/dist
+
```

Comment:
I would add also here package-lock.json, so it won't be committed

Reply:
In other projects I have always left package-lock.json in the repo as dependency issues can arise when the repo is cloned and npm install is run locally, but willing to explore the issue more @mcarmenjc @DarraghOF